include/quicr/detail/messages.h

```diff
@@ -64,6 +65,7 @@ namespace quicr::messages {
 
         STREAM_HEADER_TRACK = 0x50,
         STREAM_HEADER_GROUP,
+        STREAM_HEADER_SUBGROUP = 0x4,
```

Comment:
Prefer keeping the 2 digit format of these numbers for clarity (0x0 being the exception)
```suggestion
        STREAM_HEADER_SUBGROUP = 0x04,
```